### PR TITLE
[installer]: change ownership

### PR DESCRIPTION
**Pull request by MrSimonEmms**
## Description

Now it's owned by the self-hosted team

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **mrsimonemms** after the PR has been reviewed.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[installer/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/installer/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
The following users are mentioned in OWNERS file(s) but are untrusted for the following reasons. One way to make the user trusted is to add them as [members]() of the gitpod-io org. You can then trigger verification by writing `/verify-owners` in a comment.
- engineering-self-hosted
  - User is not a member of the org. User is not a collaborator. Satisfy at least one of these conditions to make the user trusted.

**Comment by svenefftinge** [comment]
Why jointly? I think it's better to have a clear responsibility (i.e. self-hosted) @csweichel any thoughts on this?

**Comment by csweichel** [comment]
> Why jointly? I think it's better to have a clear responsibility (i.e. self-hosted) @csweichel any thoughts on this?

Agreed with Sven. The ownership of the installer should move entirely to the self-hosted team.

**Comment by roboquat** [comment]
The following users are mentioned in OWNERS file(s) but are untrusted for the following reasons. One way to make the user trusted is to add them as [members]() of the gitpod-io org. You can then trigger verification by writing `/verify-owners` in a comment.
- engineering-self-hosted
  - User is not a member of the org. User is not a collaborator. Satisfy at least one of these conditions to make the user trusted.

**Comment by MrSimonEmms** [comment]
Ok, change made. The self-hosted team will need to be verified as part of the engineering team first (@corneliusludmann added it this morning)

**Review by princerachit — APPROVED** [review]

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: a7aef2d934dedf07a73655311e44e2f24dcf2a73</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7199#pullrequestreview-831153801" title="Approved">princerachit</a>*

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[installer/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/installer/OWNERS)~~ [princerachit]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7199#issuecomment-993332574" title="LGTM">corneliusludmann</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/7199#pullrequestreview-831153801" title="Approved">princerachit</a>*

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[installer/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/installer/OWNERS)~~ [princerachit]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
The following users are mentioned in OWNERS file(s) but are untrusted for the following reasons. One way to make the user trusted is to add them as [members]() of the gitpod-io org. You can then trigger verification by writing `/verify-owners` in a comment.
- engineering-self-hosted
  - User is not a member of the org. User is not a collaborator. Satisfy at least one of these conditions to make the user trusted.

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7199#issuecomment-993332892" title="Approved">corneliusludmann</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/7199#pullrequestreview-831153801" title="Approved">princerachit</a>*

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[installer/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/installer/OWNERS)~~ [princerachit]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
The following users are mentioned in OWNERS file(s) but are untrusted for the following reasons. One way to make the user trusted is to add them as [members]() of the gitpod-io org. You can then trigger verification by writing `/verify-owners` in a comment.
- engineering-self-hosted
  - User is not a member of the org. User is not a collaborator. Satisfy at least one of these conditions to make the user trusted.

**Comment by roboquat** [comment]
The following users are mentioned in OWNERS file(s) but are untrusted for the following reasons. One way to make the user trusted is to add them as [members]() of the gitpod-io org. You can then trigger verification by writing `/verify-owners` in a comment.
- engineering-self-hosted
  - User is not a member of the org. User is not a collaborator. Satisfy at least one of these conditions to make the user trusted.

**Comment by roboquat** [comment]
New changes are detected. LGTM label has been removed.

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7199#issuecomment-993332892" title="Approved">corneliusludmann</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/7199#pullrequestreview-831153801" title="Approved">princerachit</a>*

Associated issue requirement bypassed by: *<a href="https://github.com/gitpod-io/gitpod/pull/7199#issuecomment-993332892" title="Approved">corneliusludmann</a>*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)~~ [corneliusludmann,princerachit]
- ~~[installer/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/installer/OWNERS)~~ [princerachit]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
The following users are mentioned in OWNERS file(s) but are untrusted for the following reasons. One way to make the user trusted is to add them as [members]() of the gitpod-io org. You can then trigger verification by writing `/verify-owners` in a comment.
- rl-gitpod
  - User is not a member of the org. User is not a collaborator. Satisfy at least one of these conditions to make the user trusted.

File: OWNERS_ALIASES
Comment (corneliusludmann) [inline OWNERS_ALIASES:34]: Let's add Chad `metcalfc` here as well.

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 2047422737b956d268b332fe8fc50ed04f9c1654</details>

**Pull request merged**
